pkg/kubelet/cm/dra/manager.go

```diff
@@ -108,11 +108,11 @@ func (m *ManagerImpl) PrepareResources(pod *v1.Pod) error {
 			continue
 		}
 
-		claimInfo := m.cache.get(*claimName, pod.Namespace)
-		if claimInfo == nil {
+		claimInfoCopy, err := m.cache.getClaimInfo(*claimName, pod.Namespace)
+		if err != nil {
```

Comment:
I don't think this rename is necessary. The new samantics of the `get` call make it clear that its a copy.

Reply:
I'd agree if we rename `get` -> `getCopy` to emphasise it even more.

Reply:
Regardless of the function name change, I still think this rename of the variable is unnecessary